### DISCUSSION TITLE
Could Game Translation Be Possible?

### DISCUSSION THREAD

**Discussion by ACS-resources** [nroutasuo/level13#263] [General]
I'm from a country that doesn't speak English, and I was hoping that the game can release translations to other frequently-used languages like French, Chinese, Germany, Portuguese. I'm not having problems reading English myself, but I still wish that the developers can do it to attract players from other countries, and it would be really great. 

Anyway, thanks a **lot** for presenting us such a great game.

**Comment by nroutasuo** [comment]
Thank you for the feedback! :) I would love to see the game translated one day but it's going to be a big project and hasn't been a priority yet. It will become more relevant once the game gets out of alpha.

**Reply by ACS-resources** [reply]
Actually it would be possible if you can use string-packs (something that's already in use by other famous incremental games like Evolve),  which can make it much easier for contributers to translate. There are existing translation to, for example Chinese, but that edition is really strange, as there are many untranslated parts and mistakes. Lastly, good luck with your development :)

**Reply by nroutasuo** [reply]
I've seen the Chinese version, are there others I'm not aware of? Either way I'll take a look at how translation is done in Evolve, thanks for the tip!

**Comment by ACS-resources** [comment]
> I've seen the Chinese version, are there others I'm not aware of? Either way I'll take a look at how translation is done in Evolve, thanks for the tip!

It's like a JSON file with a format like
```json
{
  "tab_evolve": "Evoluce",
  "tab_city1": "Kemp",
  "tab_city2": "Osada",
  "tab_city3": "Vesnice",
  "tab_city4": "Městečko",
  "tab_city5": "Město",
  "tab_city6": "Metropole",
  "tab_city7": "Megalopole",
  "tab_city8": "Ekumenopole",
  "tab_civil": "Civilizace",
  "tab_space": "Vesmír",
  "tab_civics": "Obyvatelé",
  "tab_gov": "Vláda",
  "tab_industry": "Průmysl",
  "tab_military": "Armáda",
  "tab_research": "Výzkum",
  "tab_resources": "Zásoby",
}
```
somehow like the | item_id -> displayed name(translation) | stuff in Minecraft

**Comment by ACS-resources** [comment]
@nroutasuo 
I'm just wondering where's the display texts located in the source code files?
If possible I'd like to make a thorough translation

**Reply by nroutasuo** [reply]
Currently the texts are spread out all over the place unfortunately. I've been meaning to clean up and put them all in a json file like we discussed earlier, but just haven't found the time yet. Also, many of the texts such as sector descriptions and log messages are composed from patterns in a way that might not be easy to replicate in other languages without some code changes (for example if you have noun genders).

**Reply by MoPepettes** [reply]
Hi there, thanks for your game. I was looking for those texts too, too bad we can't translate.
